src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,48 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+function findUser(userId: number, users: User[]): User {
+  const findedUser = users.find(user => user.id === userId);
+
+  return !findedUser
+    ? {
```

Comment:
Returning a hardcoded user object when no user is found might not be the best approach since it obscures the fact that a user was not found. It may be better to return null and handle this case appropriately in the rendering logic.